doc/changelog/03-notations/12218-numeral-notations-non-inductive.rst

```diff
@@ -0,0 +1,19 @@
+- **Deprecated**
+  ``Numeral.v``, please use ``Number.v`` instead.
+- **Changed**
+  the way rational and real constants are parsed.
```

Comment:
I had remembrance of @Zimmi48 telling me the contrary but couldn't find the message again, so my memory might well be fooling me.
Anyway, although both styles appear in recent changes, complete sentences are far more common, so i'l perform your suggested change, if not told otherwise.

Reply:
I think this change request is still pending (and I'm starting another stab at reviewing, though I may not finish tonight)